cms/templates/group_configurations.html

```diff
@@ -47,32 +49,45 @@ <h1 class="page-header">
 
   <div class="wrapper-content wrapper">
     <section class="content">
+
       <article class="content-primary" role="main">
-        <div class="wrapper-groups content-groups">
-          <h3 class="title">${_("Content Groups")}</h3>
-            <div class="ui-loading">
-                <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
-            </div>
-        </div>
 
-      % if should_show_experiment_groups:
-        <div class="wrapper-groups experiment-groups">
-          <h3 class="title">${_("Experiment Group Configurations")}</h3>
-          % if experiment_group_configurations is None:
-            <div class="notice notice-incontext notice-moduledisabled">
-                <p class="copy">
-                    ${_("This module is disabled at the moment.")}
-                </p>
-            </div>
-          % else:
-            <div class="ui-loading">
-                <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
-            </div>
-          % endif
-        </div>
-      % endif
+        % for config in all_group_configurations:
+          <div class="wrapper-groups content-groups ${config['scheme']}">
+            <h3 class="title">${config['name']}</h3>
+              <div class="ui-loading">
+                  <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
+              </div>
+          </div>
+        % endfor
+
+        % if should_show_experiment_groups:
+          <div class="wrapper-groups experiment-groups">
+            <h3 class="title">${_("Experiment Group Configurations")}</h3>
+            % if experiment_group_configurations is None:
+              <div class="notice notice-incontext notice-moduledisabled">
+                  <p class="copy">
+                      ${_("This module is disabled at the moment.")}
+                  </p>
+              </div>
+            % else:
+              <div class="ui-loading">
+                  <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
+              </div>
+            % endif
+          </div>
+        % endif
       </article>
       <aside class="content-supplementary" role="complementary">
+      % if should_show_enrollment_track:
+        <div class="bit">
+          <div class="enrollment-track-doc">
+            <h3 class="title-3">${_("Enrollment Track Content Groups")}</h3>
```

Comment:
@staubina Revised:
Enrollment Track Groups

Enrollment track groups allow you to offer different course content to learners in each enrollment track. Learners enrolled in each enrollment track in your course are automatically included in the corresponding enrollment track group.

On unit pages in the course outline, you can designate components as visible only to learners in a specific enrollment track.

You cannot edit enrollment track groups, but you can expand each group to view details of the course content that is designated for learners in the group.